jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -114,6 +118,38 @@ public static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	private String checkPointDir;
+	private boolean keepRunning = false;
+	private boolean shellJob = true;
+	private boolean extUnixSupport = true;
+	private int logLevel = 4;
+	private String logFile = "checkpoint.log";
+
+	public CRIUSupport setCheckPointDir(String checkPointDir) {
```

Comment:
this should nullcheck the path and also perform a security check, then you can remove the security check in `checkPointJVM`

Reply:
Also, please rename these to match the criu names. So this should be `setImageDir`